fanficfare/adapters/adapter_royalroadcom.py

```diff
@@ -128,8 +128,8 @@ def make_soup(self, data):
         style_elements = soup.find_all('style')
         for style_element in style_elements:
             style_content = style_element.string
-            if style_content and 'display: none;' in style_content.lower():
-                class_matches = re.findall(r'\.(\S+)\s*\{\s*display:\s*none;\s*\}', style_content)
+            if style_content and 'display: none;' in style_content:
```

Comment:
Removing `.lower()` from the initial check basically negates adding `flags=re.IGNORECASE` to the class parser.  

I think @JasonGross's point was that `'display: none;' in style_content` is checked before the regex, so it either needs to be similarly widened, or better, removed in favor of checking that `class_matches` exists, indicating regex match.

Reply:
Like this maybe ?